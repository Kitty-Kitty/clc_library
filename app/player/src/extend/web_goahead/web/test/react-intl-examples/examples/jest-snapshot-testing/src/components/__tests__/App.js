/*global expect*/

import React from 'react';
import createComponentWithIntl from '../../utils/createComponentWithIntl';
import App from '../App';

test('app should be rendered', () => {
    const component = createComponentWithIntl(<App/>);

    let tree = component.toJSON();

    expect(tree).toMatchSnapshot();
});
