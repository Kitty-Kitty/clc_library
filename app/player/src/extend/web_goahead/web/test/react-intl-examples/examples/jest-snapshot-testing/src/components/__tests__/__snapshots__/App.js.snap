exports[`test app should be rendered 1`] = `
<div
  className="app">
  <header
    className="app-header">
    <div
      className="app-header__title">
      <span>
        React Intl + Jest Snapshot Testing Example
      </span>
    </div>
  </header>
  <main
    className="app-main"
    onClick={[Function]}>
    <div
      className="app-main__counter">
      <span>
        Counter 1
      </span>
    </div>
  </main>
  <footer
    className="app-footer">
    <div
      className="app-footer__copyright">
      <span>
        Â© 2017 Future Testing
      </span>
    </div>
  </footer>
</div>
`;
